src/components/dialog/dialog.style.js

```diff
@@ -72,7 +73,6 @@ const DialogStyle = styled.div`
       min-height: 0px !important;
     `}
 
-
   ${StyledIconButton} {
```

Comment:
I think a better fix would be to just move this `StyledIconButton` css into `DialogTitleStyle` then it'll only apply to the Dialog close icon as intended. It shouldn't be applying to anything inside the Dialog content

Then they'll be no need for the overriding CSS specifically for Pill below, we'll be removing styles from Pill rather than adding even more

Reply:
Good idea. I did try moving the code around to avoid having to add more code to the styles but couldn't achieve what was required in the ticket. I'll give your suggestion a shot and see what is what. Thanks.